worker/respec-worker.js

```diff
@@ -10,20 +10,45 @@ try {
   console.error("Network error loading/configuring highlighter", err);
 }
 
+try {
+  importScripts("https://www.w3.org/Tools/respec/webidl2.js");
```

Comment:
I think this may cause test failure when upgrading webidl2.js. Not sure what would be the best way though...

Reply:
ah, true. We might need to signal debug mode somehow.

Reply:
webpack has a solution specifically for workers, maybe we can do a similar thing. https://github.com/webpack-contrib/worker-loader